src/wallet/wallet.cpp

```diff
@@ -2149,10 +2149,10 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const int nConfMin
 
     if (nTotalLower == nTargetValue)
     {
-        for (unsigned int i = 0; i < vValue.size(); ++i)
+        for (const auto& inputCoin : vValue)
```

Comment:
Simply call this `input` or `coin`, to avoid making it look like Hungarian notation for a transaction input?